benchmarking/evaluations/metrics/utils/student_calculations.py

```diff
@@ -21,7 +21,7 @@ def num_enemies_satisfied(student: Student) -> int:
             continue
         count += 1
         for teammate in student.team.students:
```

Comment:
similar down here it's weird to add 1 and then remove 1 if they are present imo, just only add one if they are not present

Reply:
```py
if relation_student_id not in [t.id for t in student.team.students if t.id != student.id]:
    count += 1
```